src/modm/platform/pwm/sam_x7x/pwm.cpp.in

```diff
@@ -0,0 +1,14 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2023, Christopher Durand
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+// source file to compile header content in HAL CI
+#include <modm/platform/pwm/pwm_{{ id }}.hpp>
```

Comment:
But all these header files are included in `main.cpp` via `<modm/platform.hpp>` anyways? The [file is generated here](https://github.com/modm-io/modm/blob/develop/src/modm/platform/module.lb#L29-L30).